pkg/scripts/os.go

```diff
@@ -25,295 +25,6 @@ const (
-	conntrack-tools \
-	ebtables \
-	socat \
-	iproute-tc
-
-{{ if .INSTALL_DOCKER }}
-{{ template "docker-daemon-config" . }}
-{{ template "yum-docker-ce-amzn" . }}
-{{ end }}
-
-{{ if .INSTALL_CONTAINERD }}
-sudo yum install runc -y
-{{ template "containerd-github" . }}
-{{ end }}
-
-sudo mkdir -p /opt/bin /etc/kubernetes/pki /etc/kubernetes/manifests
-
-rm -rf /tmp/k8s-binaries
-mkdir -p /tmp/k8s-binaries
-cd /tmp/k8s-binaries
-
-{{- if .CNI_URL }}
-sudo mkdir -p /opt/cni/bin
-curl -L "{{ .CNI_URL }}" | sudo tar -C /opt/cni/bin -xz
-{{- end }}
-
-{{- if .NODE_BINARIES_URL }}
-curl -L --output /tmp/k8s-binaries/node.tar.gz {{ .NODE_BINARIES_URL }}
-tar xvf node.tar.gz
-{{- end }}
-
-{{- if and .KUBELET .NODE_BINARIES_URL }}
-sudo install --owner=0 --group=0 --mode=0755 /tmp/k8s-binaries/kubernetes/node/bin/kubelet /opt/bin/kubelet
-sudo ln -sf /opt/bin/kubelet /usr/bin/
-rm /tmp/k8s-binaries/kubernetes/node/bin/kubelet
-
-cat <<EOF | sudo tee /etc/systemd/system/kubelet.service
-[Unit]
-Description=kubelet: The Kubernetes Node Agent
-Documentation=https://kubernetes.io/docs/home/
-Wants=network-online.target
-After=network-online.target
-
-[Service]
-ExecStart=/opt/bin/kubelet
-Restart=always
-StartLimitInterval=0
-RestartSec=10
-
-[Install]
-WantedBy=multi-user.target
-EOF
-
-sudo mkdir -p /etc/systemd/system/kubelet.service.d
-cat <<EOF | sudo tee /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
-[Service]
-Environment="KUBELET_KUBECONFIG_ARGS=--bootstrap-kubeconfig=/etc/kubernetes/bootstrap-kubelet.conf --kubeconfig=/etc/kubernetes/kubelet.conf"
-Environment="KUBELET_CONFIG_ARGS=--config=/var/lib/kubelet/config.yaml"
-# This is a file that "kubeadm init" and "kubeadm join" generates at runtime, populating the KUBELET_KUBEADM_ARGS variable dynamically
-EnvironmentFile=-/var/lib/kubelet/kubeadm-flags.env
-# This is a file that the user can use for overrides of the kubelet args as a last resort. Preferably, the user should use
-# the .NodeRegistration.KubeletExtraArgs object in the configuration files instead. KUBELET_EXTRA_ARGS should be sourced from this file.
-EnvironmentFile=-/etc/default/kubelet
-ExecStart=
-ExecStart=/opt/bin/kubelet \$KUBELET_KUBECONFIG_ARGS \$KUBELET_CONFIG_ARGS \$KUBELET_KUBEADM_ARGS \$KUBELET_EXTRA_ARGS
-EOF
-{{- end }}
-
-{{- if and .KUBEADM .NODE_BINARIES_URL }}
-sudo install --owner=0 --group=0 --mode=0755 /tmp/k8s-binaries/kubernetes/node/bin/kubeadm /opt/bin/kubeadm
-sudo ln -sf /opt/bin/kubeadm /usr/bin/
-rm /tmp/k8s-binaries/kubernetes/node/bin/kubeadm
-{{- end }}
-
-{{- if and .KUBECTL .KUBECTL_URL }}
-curl -L --output /tmp/k8s-binaries/kubectl {{ .KUBECTL_URL }}
-sudo install --owner=0 --group=0 --mode=0755 /tmp/k8s-binaries/kubectl /opt/bin/kubectl
-sudo ln -sf /opt/bin/kubectl /usr/bin/
-rm /tmp/k8s-binaries/kubectl
-{{- end }}
-
-sudo systemctl daemon-reload
-sudo systemctl enable --now kubelet
-
-{{- if or .FORCE .KUBELET }}
-sudo systemctl restart kubelet
-{{- end }}
-`
-
 	kubeadmCoreOSTemplate = `
```

Comment:
Can you also move CoreOS/Flatcar templates to a dedicated file for consistency?

Reply:
I will actually move this extraction to the dedicated PR, as this move is not strictly related to the safeguard :)